pkg/cmd/job/job_show_runs.go

```diff
@@ -27,9 +27,9 @@ func newCmdJobShowRuns(ctx api.Context) *cobra.Command {
 
 			var runs []metronome.Run
 			if runID != "" {
-				run, err := client.Run(args[0], runID)
-				if err != nil {
-					return err
+				run, runErr := client.Run(args[0], runID)
```

Comment:
Why can't we use `err` here? Looks like a false-positive with variable shadowing. Especially for errors I think we should default to the idiomatic `err`, and only have different variable names when they need to be kept for later usage, which should be egde-cases.

Reply:
It shadows `err` from above

Reply:
My point is that it's a false-positive. As errors are usually checked only once and right after they're declared, it is fine to have `err` here and I think it's idiomatic Go.

Even when I vet the Go source code itself for shadowing there are hundreds of errors like `src/archive/tar/reader.go:81:6: declaration of "err" shadows declaration at line 77`.

Reply:
> My point is that it's a false-positive.

How it could be a false positive if it shadows previous declaration

> Even when I vet the Go source code itself for shadowing there are hundreds

Go code if full of some practices we do not follow. Take a look at @makkes comments on `e` error name. Go code is full of single variable names, or weird 3 letters words that are hard to decrypt. Do we also want to follow that rule? If you do not like this check we can disable it but IMO it's better to have errors with clear names indicating what they are used for.

Reply:
> How it could be a false positive if it shadows previous declaration

It is a false positive because the `err` variable of the outer block is not meant to be changed here.

Furthermore, the outer `err` variable is referenced below in the else block only, so I'd even argue that it is shadowing at all here, there is no code path going through the inner and outer `err` variable afterwards.

I'd rather remove this rule, see https://github.com/golang/go/issues/19490#issuecomment-285581930 where they say it is known to give many false-positives and still experimental.

Reply:
I disabled shadow check